components/integration/replace-apollo-graphql/code/apollo-graphql-bbe/subscription-support.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Out-of-the-box subscription support - No additional libraries"
+description: "Ballerina offers seamless support for GraphQL subscriptions out-of-the-box, eliminating the need for additional libraries like with Apollo. With Ballerina, you can effortlessly integrate systems like Apache Kafka into your GraphQL subscriptions, enhancing real-time data streaming capabilities. This simplifies development, reduces dependencies, and provides a comprehensive solution for building robust GraphQL subscription-based applications."
+url: "https://github.com/ballerina-platform/module-ballerina-graphql/tree/master/examples/news_alerts"
+---
+
+```
+import ballerina/graphql;
+import ballerina/log;
+import ballerina/uuid;
+import ballerinax/kafka;
```

Comment:
Let's remove the imports to reduce the line count , we did the same in integration page.

Reply:
Updated the code just to show the required parts. 13c6c94aa68f7344aa95ee4fb140d3d6991e170f